auto3dseg/algorithm_templates/dints/scripts/search.py

```diff
@@ -520,9 +525,7 @@ def run(config_file: Optional[Union[str, Sequence[str]]] = None, **override):
                                 mode="gaussian",
                                 overlap=overlap_ratio,
                                 sw_device=device)
-                    except RuntimeError as e:
-                        if not any(x in str(e).lower() for x in ("memory", "cuda", "cudnn")):
-                            raise e
+                    except BaseException:
```

Comment:
Using a `BaseException` without triaging could make the debug difficult in the future

Reply:
I have updated the exception.